packages/core/src/Modules/Cashier/Containers/p2p-cashier.jsx

```diff
@@ -54,6 +54,7 @@ class P2PCashier extends React.Component {
                 setNotificationCount={setNotificationCount}
                 order_id={order_id}
                 setOrderId={this.setQueryOrder}
+                poi_url='/account/proof-of-identity'
```

Comment:
Can we use the routes constant available from `'@deriv/shared/utils/routes'`?

Reply:
yes, thank you 👍